infra/Pos/Reporting/Methods.hs

```diff
@@ -118,23 +117,20 @@ ipv4Local w =
 
 -- | Retrieves node info that we would like to know when analyzing
 -- malicious behavior of node.
-getNodeInfo :: (MonadDHT m, MonadIO m) => m Text
+getNodeInfo :: (MonadIO m) => m Text
```

Comment:
https://issues.serokell.io/issue/CSL-1011

`reportMisbehaviour` is a more involved issue.

For now, I'm just going to throw the `getPeers` parameter everywhere it's needed.